# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../portmap/perror.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./portmap.c.orig	Fri May 31 15:52:59 1996
+++ ./portmap.c	Fri Apr 23 10:38:46 1999
@@ -235,7 +235,7 @@
 /* need to override perror calls in rpc library */
 void
 perror(what)
-	const char *what;
+	__const char *what;
 {
 
 	syslog(LOG_ERR, "%s: %m", what);
